dttools/src/link.c

```diff
@@ -435,23 +435,23 @@ struct link *link_serve_address(const char *addr, int port)
 	return 0;
 }
 
-struct link *link_accept(struct link *master, time_t stoptime)
```

Comment:
This is referring to the original TCP connection that listens on a specific port, so `manager` isn't the right term here.  Maybe `parent` or `fixed`?